crane-cloud/backend
# User collaboration url

# Description
This fixes URL for the user collaboration invite email in staging and production.

## Type of change


## Trello Ticket ID
https://trello.com/c/w83A78OG/1526-user-collaboration-url-for-staging-and-production

## How Can This Be Tested?
Check out the branch and navigate to the endpoint to invite users for project collaboration. Assign a user to an existing project by filling in the project_id, email, and assigning a role to the user (member or admin). The URL received in the email denotes the host address inviting for the collaboration.

## Checklist:
